### ISSUE TITLE
Split issue in the ford dataset

### ISSUE THREAD

**Issue by tho-kn** [ShanWang-Shan/PureACL#3]
https://github.com/ShanWang-Shan/PureACL/blob/28d634d9488a92abe13680176d7eb033044cde21/PureACL/pixlib/datasets/ford.py#L156-L161

Currently dataloader uses test set directory for training, L158 should be elif

**Comment by ShanWang-Shan** [comment]
Thanks, modified

**Comment by vavisc** [comment]
Does this imply that the results reported in the paper, particularly in scenarios using the Ford dataset, may not be fully valid? If the test data was used during training, this could lead to overly optimistic performance metrics.
